src/components/Signup/Signup.js

```diff
@@ -153,6 +153,7 @@ const Signup = function ({
 
     const result = await dopplerLegacyClient.registerUser({
       ...values,
+      email: values[fieldNames.email].trim(),
```

Comment:
Did you test forgot password? because it happened there also according with the ticket.

Reply:
You're right!